dev/ci/unicoq-num.patch

```diff
@@ -0,0 +1,44 @@
+commit f29bc64ee3d8b36758d17e1f5d50812e0c93063b
```

Comment:
What stops this patch from going upstream?

Reply:
Nobody seems competent to write a proper install script for Windows:
https://github.com/unicoq/unicoq/pull/17#issuecomment-442302813

Reply:
That PR is merged and seems to do the same thing as the patch, so why do we need the patch?

Reply:
IIRC it's been reverted.